api/v1alpha1/shared_types.go

```diff
@@ -21,6 +21,14 @@ const (
 	DefaultDeploymentMemoryResourceRequests = "512Mi"
 	// DefaultEnvoyProxyImage is the default image used by envoyproxy
 	DefaultEnvoyProxyImage = "envoyproxy/envoy:distroless-dev"
+	// DefaultShutdownManagerCPUResourceRequests for shutdown manager cpu resource
+	DefaultShutdownManagerCPUResourceRequests = "10m"
+	// DefaultShutdownManagerMemoryResourceRequests for shutdown manager memory resource
+	DefaultShutdownManagerMemoryResourceRequests = "32Mi"
+	// DefaultShutdownManagerImageName is the default image used for the shutdown manager.
+	DefaultShutdownManagerImageName = "envoyproxy/gateway-dev"
```

Comment:
can we combine these two fields ? similar to `DefaultEnvoyProxyImage` ?

Reply:
@arkodg That's how it originally was, see here for why: https://github.com/envoyproxy/gateway/pull/2633#discussion_r1503008338

I'm ok with either approach tbh, we just have to pick one.

Reply:
ah I just realized the conundrum you're in, big +1 to the approach taken using the ldflags !